### ISSUE TITLE
Stab hat keine Volumenbegrenzung

### ISSUE THREAD

**Issue by JulianSchneider** [Elytherion/tdeheroes#278]
Stäbe habe einen Volumenbegrenzung von Stabzaubern, man kann aber trotzdem fröhlich weiter Zauber hinzufügen.

**Comment by Elytherion** [comment]
Ja, das stimmt. Das Feature wird aber schon teilweise über #273 getrackt. Ich ändere mal den Titel, damit das hier nur für pAsP gilt, was noch nicht in einem Issue drin ist, meine ich.

**Comment by Elytherion** [comment]
Was ich noch vergessen habe zu erwähnen: Das Kaufen von Stabzaubern oder anderen Traditionsartefakt-SFs impliziert nicht deren Ausführung. Du kannst _Bindung des Stabes_ haben, aber keinen eigenen Magierstab. Mit der SF erlernst du die Fähigkeit, das Ritual auszuführen. Ob man es dann einsetzt, steht auf einem anderen Blatt. Man kann ja auch z.B. über eine erneute _Bindung des Stabes_ einen neuen Stab an sich binden &ndash; und muss nicht die SF erneut kaufen.

**Comment by JulianSchneider** [comment]
Tatsächlich, habe ich einen Malus bekommen für meine Stabzauber:
![image](https://user-images.githubusercontent.com/15930333/62686085-4d1d3880-b9c4-11e9-8a9b-d7dd2b24b153.png)

Ist das so korrekt?

**Comment by Elytherion** [comment]
Wo ist denn da der Malus? 🤔 Wenn du einen Malus an pAsP hättest, wären sie unten in den kleinen Kästchen aufgelistet. Da steht aber bei beiden 0.

**Comment by JulianSchneider** [comment]
Jo, ist ein Bonus, habe ich auch gerade erst verstanden, nvm :)
